backend/src/db/migrations/20200206190233-swap_latitude_with_longitude.js

```diff
@@ -0,0 +1,42 @@
+import { getDriver } from '../../db/neo4j'
+
+export const description = `
+This migration swaps the value stored in Location.lat with the value
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-02-14T11:31:12Z" title="Friday, February 14th 2020, 12:31:12 pm +01:00">Feb 14, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,37 @@
+import { getDriver } from '../../db/neo4j'
+
+export const description =
+  'This migration swaps the value stored in Location.lat with the value of Location.lng.'
```


I would love a better description for why this was necessary, especially for our future selves/other programmers to understand why you would ever want to swap these values